_maps/map_files/SolitaireStation/Solitairestation.dmm

```diff
@@ -66648,7 +66650,8 @@
 /area/command/heads_quarters/hop)
 "tqQ" = (
 /obj/machinery/door/airlock/maintenance{
-	req_access_txt = "12"
+	name = "Service Maintenance Access";
+	req_one_access_txt = "12;22;25;26;28;35;37;38;46;70"
```

Comment:
read above

Reply:
> read above

ASSHOLE i didnt do anything i copy pasted the service door on helio and switched it with the maint door on solitare

Reply:
https://prnt.sc/26maugs

Reply:
"i used a bad station and the design was bad" cry about it liberal